interfaces/builtin/maliit.go

```diff
@@ -122,34 +124,36 @@ func (iface *MaliitInterface) PermanentPlugSnippet(plug *interfaces.Plug, securi
 }
 
 func (iface *MaliitInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
-	switch securitySystem {
-	case interfaces.SecurityAppArmor:
-		old := []byte("###SLOT_SECURITY_TAGS###")
-		new := slotAppLabelExpr(slot)
-		snippet := bytes.Replace([]byte(maliitConnectedPlugAppArmor), old, new, -1)
-		return snippet, nil
-	}
 	return nil, nil
 }
 
+func (iface *MaliitInterface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.Plug, slot *interfaces.Slot) error {
+	old := "###SLOT_SECURITY_TAGS###"
+	new := slotAppLabelExpr(slot)
+	snippet := strings.Replace(maliitConnectedPlugAppArmor, old, new, -1)
+	return spec.AddSnippet(snippet)
+}
+
+func (iface *MaliitInterface) SecCompPermanentSlot(spec *seccomp.Specification, slot *interfaces.Slot) error {
+	return spec.AddSnippet(maliitPermanentSlotSecComp)
```

Comment:
Yes. This interface landed last Friday and was missing the new methods for seccomp; in fact its partially broken (the seccomp part of it) in master because of this.